src/profiling/with_perf.md

```diff
@@ -1,17 +1,18 @@
 # Profiling with perf
 
-sThis is a guide for how to profile rustc with perf.
+This is a guide for how to profile rustc with [perf](https://perf.wiki.kernel.org/index.php/Main_Page).
 
 ## Initial steps
 
-- Get a clean checkout of rust-lang/master, or whatever it is you want to profile.
+- Get a clean checkout of rust-lang/master, or whatever it is you want
+  to profile.
 - Set the following settings in your `config.toml`:
   - `debuginfo-lines = true`
-  - `use-jemalloc = false` -- lets you do memory use profiling with valgrind 
+  - `use-jemalloc = false` — lets you do memory use profiling with valgrind 
   - leave everything else the defaults
 - Run `./x.py build` to get a full build
 - Make a rustup toolchain (let's call it `rust-prof`) pointing to that result
-  - `rustup toolchain link` XXX
+  - `rustup toolchain link <path-to-toolchain>`
```

Comment:
I think this is missing the toolchain name, right? Perhaps we can just link to `how-to-build-and-run.html#creating-a-rustup-toolchain`?